python/ray/node.py

```diff
@@ -683,3 +733,108 @@ def remaining_processes_alive(self):
             True if any process that wasn't explicitly killed is still alive.
         """
         return not any(self.dead_processes())
+
+
+class LocalNode(object):
```

Comment:
Please also add docstring for `LocalNode` as `Node` do. At first glance, this class looks a bit strange. Does `LocalNode` shares the same interface with `Node`?